profile/README.md

```diff
@@ -2,13 +2,13 @@
 
 ### #️⃣ About #paid
 
-[#paid](https://www.hashtagpaid.com?utm_source=github.com&utm_medium=referral&utm_campaign=org-page) is a creator marketing platform that sits at the intersection of brand and creator collaboration. We help direct-to-consumer and Fortune 500 brands like Phillips Hue, Everlywell, Unilever, and Sephora, activate creators to drive growth.
+[#paid](https://www.hashtagpaid.com?utm_source=github.com&utm_medium=referral&utm_campaign=org-page) is a creator marketing platform that sits at the intersection of brand and creator collaboration. We help direct-to-consumer and Fortune 500 brands like Infiniti, TJX, Unilever, and Sephora, activate creators to drive growth.
```

Comment:
<img width="1409" alt="Screenshot 2024-02-07 at 2 40 31 PM" src="https://github.com/hashtagpaid/.github/assets/1641464/0c82414f-1ec1-4eb3-a2fd-642a7ea6c11a">

Reply:
can we use TJX?

Reply:
@billxinli yep, i believe so. I'm using the list of companies currently mentioned in the JDs [here](https://hashtagpaid.com/people#opportunities).

Reply:
sounds good